samples/cpp/tutorial_code/core/out_of_focus_deblur_filter/out_of_focus_deblur_filter.cpp

```diff
@@ -0,0 +1,141 @@
+﻿/**
+* @file out_of_focus_deblur_filter.cpp
+* @brief You will learn how to recover an out-of-focus image by Wiener filter
+* @author Karpushin Vladislav, karpushin@ngs.ru, https://github.com/VladKarpushin
+*/
+#include "opencv2/opencv.hpp"
```

Comment:
`opencv2/opencv.hpp` is old compatibility header (deprecated for new code including OpenCV code itself). It is not actual anymore due huge amount of OpenCV modules.

Use headers from used OpenCV modules directly:
- opencv2/core.hpp
- opencv2/imgcodecs.hpp
- opencv2/highgui.hpp
- opencv2/imgproc.hpp

Reply:
I got it